pyproject.toml

```diff
@@ -32,24 +32,20 @@ classifiers=[
 ]
 dynamic = ["version"]
 dependencies = [
-    "importlib_metadata",
     # For proper typing, mypy needs numpy>=1.20.0 (cf. https://github.com/numpy/numpy/pull/16515)
     # Additional typing support is brought by numpy>=1.22.4, but core build sticks to >=1.16.0
     "numpy>=1.16.0,<2.0.0",
     "scipy>=1.4.0,<2.0.0",
     "h5py>=3.1.0,<4.0.0",
-    "opencv-python>=4.5.0,<5.0.0",
+    "opencv-python-headless>=4.5.0,<5.0.0",
```

Comment:
I think we should have split that PR because this change on its own could have been separated from the rest, and simply speeds up installation :+1: 

But let's see if we can make it

Reply:
Yeah i agree moved to #1558